src/output.rs

```diff
@@ -9,3 +9,37 @@ pub fn print_comparisons(format: OutputFormat, comparisons: &Comparisons) {
         OutputFormat::Markdown => markdown::print_comparisons(comparisons),
     }
 }
+fn prettify_integer(i: impl ToString) -> String {
+    let printed = i.to_string();
+
+    let tmp: Vec<char> = printed.chars().collect();
+    tmp.into_iter()
+        .rev()
+        .enumerate()
+        .rev()
+        .flat_map(|(idx, c)| {
+            if idx % 3 == 0 && c != '-' && idx != 0 {
+                [c].into_iter().chain(Some('_'))
```

Comment:
I think a space or comma would be easier to read, especially in a sans-serif font.  What do you think?

Reply:
Let's look at the variations:

# Current

| metric | value |     | change |
| ------ | ----: | :-: | :----: |
| binary-size-nk3am | 1199399 | :white_check_mark: | -8963 (-0.01%) |
| binary-size-nk3am-test | 2001105 | :red_circle: | +863 (+0.00%) |
| binary-size-nk3xn | 431492 | :white_check_mark: | -3248 (-0.01%) |
| binary-size-nk3xn-test | 584604 | :white_check_mark: | -3600 (-0.01%) |

# underscore

| metric | value |     | change |
| ------ | ----: | :-: | :----: |
| binary-size-nk3am | 1_199_399 | :white_check_mark: | -8963 (-0.01%) |
| binary-size-nk3am-test | 2_001_105 | :red_circle: | +863 (+0.00%) |
| binary-size-nk3xn | 431_492 | :white_check_mark: | -3248 (-0.01%) |
| binary-size-nk3xn-test | 584_604 | :white_check_mark: | -3600 (-0.01%) |

# underscore + monospace

| metric | value |     | change |
| ------ | ----: | :-: | :----: |
| binary-size-nk3am | `1_199_399` | :white_check_mark: | -8963 (-0.01%) |
| binary-size-nk3am-test | `2_001_105` | :red_circle: | +863 (+0.00%) |
| binary-size-nk3xn | `431_492` | :white_check_mark: | -3248 (-0.01%) |
| binary-size-nk3xn-test | `584_604` | :white_check_mark: | -3600 (-0.01%) |

# comma + monospace

| metric | value |     | change |
| ------ | ----: | :-: | :----: |
| binary-size-nk3am | `1,199,399` | :white_check_mark: | -8963 (-0.01%) |
| binary-size-nk3am-test | `2,001,105` | :red_circle: | +863 (+0.00%) |
| binary-size-nk3xn | `431,492` | :white_check_mark: | -3248 (-0.01%) |
| binary-size-nk3xn-test | `584,604` | :white_check_mark: | -3600 (-0.01%) |

# comma

| metric | value |     | change |
| ------ | ----: | :-: | :----: |
| binary-size-nk3am | 1,199,399 | :white_check_mark: | -8963 (-0.01%) |
| binary-size-nk3am-test | 2,001,105 | :red_circle: | +863 (+0.00%) |
| binary-size-nk3xn | 431,492 | :white_check_mark: | -3248 (-0.01%) |
| binary-size-nk3xn-test | 584,604 | :white_check_mark: | -3600 (-0.01%) |

Reply:
Ok comma is indeed better. The numbers themselves appear to be monospace so the width of the number is proportional to the number of characters as expected.

Reply:
I prefer comma.

Reply:
I changed to commas.